Goal: Adjustments in resmgr template

Author: Peter Eisentraut <petere@debian.org>

Upstream status: Specific to Debian

Index: b/templates/resmgr.conf.template.in
===================================================================
--- a/templates/resmgr.conf.template.in
+++ b/templates/resmgr.conf.template.in
@@ -1,5 +1,5 @@
 KOLAB_META_START
-TARGET=@resmgr_confdir@/config.php
+TARGET=/etc/kolab/filter/config.php
 PERMISSIONS=0640
 OWNERSHIP=@resmgr_conffile_usr@:@resmgr_conffile_grp@
 KOLAB_META_END
@@ -166,8 +166,8 @@
  */
 $conf['log']['enabled']          = true;
 $conf['log']['priority']         = PEAR_LOG_DEBUG; // Leave this on DEBUG for now. We just restructured the package...
-$conf['log']['type']             = 'file';
-$conf['log']['name']             = '@resmgr_logdir@/filter.log';
+$conf['log']['type']             = 'syslog';
+$conf['log']['name']             = 'cons, pid';
 $conf['log']['ident']            = 'Kolab Filter';
 $conf['log']['params']['append'] = true;
 
